Script.js

```diff
@@ -0,0 +1,634 @@
+  selectedCardArray = [];
+  IsHighCardTrue = false;
+  isOnePairTrue = false;
+  isTwoPairTrue = false;
+  isThreeOfAKindTrue = false;
+  isFourOfAKindTrue = false;
+  isFiveOfAKindTrue = false;
+  isFlushTrue = false;
+  isStraightTrue = false;
+  isStraightFlush = false;
+  isFullHouseTrue = false;
+  highcard = 0;
+  suitResult = false;
+
+  resetBtn.disabled = true;
+  dealBtn.disabled = false;
+  commentDiv.innerHTML = 'Please press deal to start the game';
+  cardDiv.classList.remove('disabledCards');
+};
+
+let gamediv;
+let buttonDiv;
+let cardDiv;
+let commentDiv;
+let totalPoints;
+let cardYouGot;
+let points = 100;
+let changeBtn;
+let dealBtn;
+let resetBtn;
+
+// method to construct the page
+// return NIL
+const init = () => {
+  gamediv = document.createElement('div');
+  gamediv.classList.add('gamediv');
+
+  const scoreDiv = document.createElement('div');
+  scoreDiv.classList.add('scoreDiv');
+  const score = document.createElement('h4');
+  score.innerHTML = 'Current Score : ';
+  totalPoints = document.createElement('h4');
+  totalPoints.classList.add('scoreDiv');
+  totalPoints.innerHTML = points;
+
+  const textContentDiv = document.createElement('div');
+  textContentDiv.classList.add('textContentDiv');
+  const textContent = document.createElement('h4');
+  textContent.innerHTML = ' You Got : ';
+  cardYouGot = document.createElement('h4');
+  cardYouGot.classList.add('textContentDiv');
+
+  buttonDiv = document.createElement('div');
+  dealBtn = document.createElement('button');
+  dealBtn.textContent = 'Deal';
+  dealBtn.addEventListener('click', dealFiveCards);
+
+  // const testBtn = document.createElement('button');
+  // testBtn.textContent = 'test';
+  // testBtn.addEventListener('click', calcHandScore);
+
+  resetBtn = document.createElement('button');
+  resetBtn.textContent = 'Next Game';
+  resetBtn.disabled = true;
+  resetBtn.addEventListener('click', reset);
+
+  cardDiv = document.createElement('div');
+  cardDiv.classList.add('card-list');
+  cardDiv.setAttribute('id', 'cardDiv');
+
+  commentDiv = document.createElement('div');
+  commentDiv.classList.add('commentDiv');
+  commentDiv.innerHTML = 'Please press deal to start the game';
+  commentDiv.style.color = 'red';
+
+  changeBtn = document.createElement('button');
+  changeBtn.textContent = 'change';
+  changeBtn.disabled = true;
+  changeBtn.addEventListener('click', change);
+
+  document.body.appendChild(gamediv);
+
+  gamediv.appendChild(scoreDiv).append(score);
+  gamediv.appendChild(scoreDiv).append(totalPoints);
+  gamediv.appendChild(textContentDiv).append(textContent);
+  gamediv.appendChild(textContentDiv).append(cardYouGot);
+
+  gamediv.appendChild(commentDiv);
+  gamediv.appendChild(buttonDiv).append(dealBtn);
+  gamediv.appendChild(buttonDiv).append(changeBtn);
+  // document.body.appendChild(buttonDiv).append(testBtn);
+  gamediv.appendChild(buttonDiv).append(resetBtn);
+  gamediv.appendChild(cardDiv);
+};
```

Comment:
consider putting these in separate helper functions so that your code is easier to read